rollup.config.js

```diff
@@ -2,51 +2,59 @@ import buble from 'rollup-plugin-buble';
 import json from 'rollup-plugin-json';
 import resolve from 'rollup-plugin-node-resolve';
 import commonjs from 'rollup-plugin-commonjs';
-import pkg from './package.json';
+const pkg = require('./package.json');
 
-const config = {
-	input: 'src/index.js',
-	plugins: [
-		json(),
-		commonjs(),
-		buble({
-			target: {
-				node: 4
-			},
-			include: [
-				'src/**',
-				'node_modules/acorn-object-spread/**',
-				'node_modules/unicode-loose-match/**',
-				'node_modules/regexpu-core/**'
-			],
-			transforms: {
-				dangerousForOf: true
-			}
-		}),
-		resolve()
-	],
-	name: 'buble',
-	sourcemap: true
+const createConfig = ({
+	browser = false,
+	external = ['acorn', 'magic-string'],
+	output,
+} = {}) => {
+	return {
+		input: 'src/index.js',
+		output,
+		external,
+		plugins: [
+			json(),
+			commonjs(),
+			buble({
+				target: !browser ? { node: 4 } : null,
```

Comment:
We could specify a target for browser as well, maybe { ie: 11 }?

Reply:
i dont think ie10 and ie11 differ that much from es5 syntax support perspective, most applications and libraries transpiles with things like `babel-preset-es2015` or `babel-preset-env` without options to target pretty much all es5 envs (syntax-wise)

So in general I wouldn't expect it to make any difference, and personally I'm treating ie10 still as sort of a baseline which I have to support (which ofc doesn't mean that you have to support it).